kubernetes-client/src/test/resources/test-template.json

```diff
@@ -1,7 +1,7 @@
 
 {
   "kind": "Template",
-  "apiVersion": "v1",
+  "apiVersion": "template.openshift.io/v1",
```

Comment:
@rohanKanojia I need your expertise here: with this change, all templates must have the `apiVersion` set to `template.openshift.io/v1`, which may break existing functionality when reading template files set to `v1`. Any suggestions or should we let it throw an error indicating that the template needs to be fixed?

Reply:
Not sure but we can try adding `notFoundTransformation` entries in `BackwardsCompatibilityInterceptor`. I think this new apiVersion compatible with openshift 3.11 and 3.10 ? I think maybe we can dump `v1` apiVersion users.....